### ISSUE TITLE
Allow resetting launch version on cartridge-based games

### ISSUE THREAD

**Issue by ghost** [XorTroll/Goldleaf#603]
# Allow resetting launch version on cartridge-based games

## Feature description

Goldleaf lets you reset the launch version for digital games, but if you select the game on cartridge slot, there is no option to reset launch version, only an "Ok" button that closes the popup. It would be nice if reset launch version could be extended to cartridge version of games.

## Benefits

Sometimes people have physical copies of games instead of digital copies.

## Additional information

![image](https://user-images.githubusercontent.com/43770697/114957874-e48c8d00-9e2f-11eb-9875-4276e2e03eeb.png)

**Comment by XorTroll** [comment]
True, I probably thought this wouldn't make sense with gamecarts, I really should fix it

**Comment by Ammako** [comment]
Also, it might be cool if cartridge games could be listed somewhere, so you could access them without needing them to be inserted. If you wanted to reset launch version, for example.

It's not a super big deal because you can just insert the cartridge anyway, but if you wanted to do operations on more than one cartridge-based game, it would be convenient to be able to access them without needing to swap cartridges every time.

**Comment by pickeditmate** [comment]
> True, I probably thought this wouldn't make sense with gamecarts, I really should fix it

Yes please do, for example the mario odyssey speed running community often need to run game version 1.0.0 to take advantage of some bugs. 

I'll be happy to test and feedback this week

**Comment by instantelement640** [comment]
i want to upgrade a version of zelda totk i have the game cartridge with version 1.0.0. installed i want to upgrade in my modded switch how do i upgrade the game software and what does resetting a launch version do in a switch in gold leaf?

**Comment by XorTroll** [comment]
Fixed in new release
